packages/cli/src/commands/serve.ts

```diff
@@ -124,6 +136,8 @@ export default class Serve extends Command {
 
     watcher.once('ready', () => {
       this.log(chalk.greenBright`Watching required files for changes .. `)
+      this.log(chalk.greenBright`Visit https://app.segment.com/dev_center/actions-tester to preview your integration.`)
+      void open('https://app.segment.com/dev_center/actions-tester')
```

Comment:
Consider this format for consistency

```suggestion
      this.log(chalk.greenBright`Visit https://app.segment.com/dev-center/actions-tester to preview your integration.`)
      void open('https://app.segment.com/dev-center/actions-tester')
```